### PR TITLE
Kb angular init

### PR DESCRIPTION
**Pull request by keithbradley1**

### CODE REVIEW

File: angular/app/albums/albumConfig.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+angular.module('We_Are_Team')
+  .config(($routeProvider) => {
+    $routeProvider
+      .when("/album", {
+        controller: "album.ctrl",
```
Comment (carter-harris) [inline angular/app/albums/albumConfig.js]: needs to match the album controller name - AlbumCtrl

File: angular/app/albums/albumConfig.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        templateUrl: "/app/album/album.html"
+      })
+      .when("/album/:albumId", {
+        controller: "albumDetailCtrl",
```
Comment (carter-harris) [inline angular/app/albums/albumConfig.js]: controller name needs to be - AlbumDetailCtrl - to match the controller

**Comment by carter-harris** [comment]
Controller names need to match the routes. 

your index has files named lowercase with capitalized camel case, 

need to look this wasy -> ArtistDetailCtrl

**Comment by Jessicashinjo** [comment]
Good starting place 👍

**Comment by carter-harris** [comment]
👍

**Pull request merged**